hamilton_launch_board/src/views/components/VideoStream.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+    <v-container>
+      <v-layout>
+        <img :src="'http://localhost:8090/' + streamFile" class="img-responsive" style="width:100%">
```

Comment:
We can get the current url.

Example from `main.js` 

```js
Vue.use(VueNativeSock, `ws://${document.location.hostname}:8000/ws`, {
```

`document.location.hostname` being the important part.